baseplate/observers/sentry.py

```diff
@@ -89,6 +106,25 @@ def error_reporter_from_config(raw_config: config.RawConfig, module_name: str) -
         else:
             break
 
+    cfg_ignore_exceptions = cfg.sentry.ignore_exceptions
+    cfg_additional_ignore_exceptions = cfg.sentry.additional_ignore_exceptions
+
+    if cfg_ignore_exceptions != None:
+        warn_deprecated(
+            "'ignore_exceptions' configuration varible is depricated."
+            + "Please use 'additional_ignore_exceptions' instead.",
```

Comment:
```suggestion
            "Please use 'sentry.additional_ignore_exceptions' instead.",
```

kinda weird but python will just auto-concatenate adjacent strings like this so we don't need a `+` here

```
>>> str("a" "b")
'ab'
```